jicofo/rootfs/defaults/jicofo.conf

```diff
@@ -187,6 +188,12 @@ jicofo {
       enable-multi-stream-backward-compat = {{ .Env.JICOFO_MULTI_STREAM_BACKWARD_COMPAT | toBool }}
       {{ end }}
 
+      {{ if $ENABLE_SHARED_DOCUMENT_RANDOM_NAME }}
+      shared-document {
+        use-random-name = {{ .Env.ENABLE_SHARED_DOCUMENT_RANDOM_NAME | toBool }}
```

Comment:
Use the variable here too, that way you don't need to use toBool.

Reply:
That's done, thanks.